backend/src/chat.gateway.ts

```diff
@@ -122,6 +122,10 @@ export class ChatGateway
         throw new WsException('Invalid Token');
       }
       this.authenticatedSockets.set(client.id, user);
+      user.memberships.forEach(
+        async (membership) =>
+          await this.handleJoin(client, { channel: membership.channel.name }),
```

Comment:
```suggestion
          await this.handleJoin(client, { channel: membership.channel.id.toString() }),
```

`handleJoin` actually takes a string of the channel's id